src/main/java/uk/gov/hmcts/probate/transformer/CaveatCallbackResponseTransformer.java

```diff
@@ -250,7 +271,8 @@ private ResponseCaveatDataBuilder getResponseCaveatData(CaveatDetails caveatDeta
             .autoClosedExpiry(caveatData.getAutoClosedExpiry())
             .pcqId(caveatData.getPcqId())
             .bulkScanEnvelopes(caveatData.getBulkScanEnvelopes())
-            .payments(caveatData.getPayments());
+            .payments(caveatData.getPayments())
+            .applicantOrganisationPolicy(caveatData.getApplicantOrganisationPolicy());
```

Comment:
Not sure if this line is needed, as you're already adding to response caveat data in transformForSolicitor() on line 164

Reply:
Thanks, I'll keep it here and remove from the transformForSolicitor - then it matches the probate code

Reply:
oh, realised I do need both, different cases - so I fill in the org details from the token in one and just keep passing it along with the other. As for probate case code.